job-simulation.rb

```diff
@@ -1,4 +1,39 @@
 require './Stack.rb'
 require './Queue.rb'
 
+class Workers
+  attr_reader :waiting, :workers
+  def initialize
+    @waiting = ten_peeps
+    @workers = six_workers
+  end
```

Comment:
This is fine, but what if we later want to change how many workers or hires we want? Can we refactor this code so that it's more scalable for other uses? Perhaps pass the ten/six as parameters instead?